.github/workflows/docker-tests.yml

```diff
@@ -1,5 +1,5 @@
 name: Continuous Build Docker
-on: [push, pull_request]
+on: [push]
```

Comment:
Does this mean forks won't build I think?

Reply:
Hmm - I investigated and believe you're right. The intention is that it should run once on every PR commit (fork or not) and on merge commits.

> By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened. To trigger workflows for more activity types, use the types keyword.


> When you create a pull request from a forked repository to the base repository, GitHub sends the pull_request event to the base repository and no pull request events occur on the forked repository.
> 
> Workflows don't run on forked repositories by default. You must enable GitHub Actions in the Actions tab of the forked repository.


So my understanding now is
- Only `pull_request` trigger works on a fork
- `pull_request` covers new commits on the pull request branch due to the `synchronize` activity type
- The `push` trigger is still needed to run this after merge, but should be filtered so it doesn't create duplicate runs on a PR.

Will raise a PR to reflect that.

Reply:
#36